### ISSUE TITLE
Question:  Experiencing Slow down

### ISSUE THREAD

**Issue by Willxiam** [nathanlesage/local-chat#3]
I was successful installing on another PC and initially I was struck by its speed, however I am experiencing slowdown despite restarting the application and the PC.   Initially it would start generating almost immediately.  Now there is sometimes as much as 30 seconds before it starts generating.  Is this typical behavior or possibly something on my end.  I will continue to experiment with the application and provide feedback.  I was happy to see the new copy functions added to conversations in the latest release.

**Comment by nathanlesage** [comment]
I noticed the same thing, and realized that there is a simple explanation: These models run on hardware that also has to run other tasks, and the models are also very large. This means that, in order to achieve the insane speeds we can see, they really have to use all the resources available.

However, if other applications require some of these resources, this means that the operating system will automatically slow down these models. For example, I noticed that generation with these models came to a screeching halt when I was simultaneously in a Zoom meeting or watching a video. Now, these problems are possibly specific to Macs, since it depends a bit on the system. If Zoom can use a dedicated hardware codec to perform the encoding/decoding of the video streams, that should not impede generation speed. However, if I'm not mistaken, the codecs on Macs are shared with the rest of the M1/2/3 cores, i.e., utilization of video en/decoding will have a performance penalty for generative AI.

In other words, depending on your system, different applications will have varying impacts on your performance. You may be able to pinpoint it by opening and closing applications and observing the generation speed.

**Comment by nathanlesage** [comment]
I'm gonna simply leave the same notice I have just added to the README here for your information:

> I noticed that LocalChat is gaining in popularity. That's great! However, this year I will have to defend my PhD thesis, so I will have limited time to code, which I will mainly direct towards my main app, [Zettlr](https://github.com/Zettlr/Zettlr). This means that LocalChat won't receive many commits from my side. That being said, **it is much easier to review & merge pull requests** and in case there are PRs that improve the app, I'll happily review and merge, and continue to update the app. However, this will only work if people open PRs. So, if you enjoy the app, please contribute. Regular development on my own side can proceed in early 2026 again.
